View3D/Content/Shaders/Phazer/MetalRoughness_main.fx

```diff
@@ -180,7 +180,14 @@ struct PixelInputType
     float3 normal : NORMAL0;
     float3 tangent : TANGENT;
     float3 binormal : BINORMAL;
-
+    
+    // TODO: REMOVE THID DEBUGGIN CODE
```

Comment:
TODO.

Reply:
I am stealing fiddling with the shaders, for debugging the imported model, so can I do this laer?

Reply:
No, a PR should reflect a finished unit of work. Hence no TODO or DEBUG stuff